lib/core/base/audit.js

```diff
@@ -165,7 +165,7 @@ Audit.prototype.run = function (context, options, resolve, reject) {
 							result: axe.constants.CANTTELL,
 							description: 'An error occured while running this rule',
 							message: err.message,
-							help: err.stack || err.message,
```

Comment:
Does anything depend on `help` being a part of this object, or was that just for manual debugging info?